### ISSUE TITLE
swaggerSchemaUIServer failure with ReqBody

### ISSUE THREAD

**Issue by defanor** [phadej/servant-swagger-ui#33]
The UI served by `swaggerSchemaUIServer` fails to issue requests whenever a JSON `ReqBody` is used, while the same code with `jensolegSwaggerSchemaUIServer` in its place works fine. Here is an example:

```haskell
{-# LANGUAGE DataKinds #-}
{-# LANGUAGE DeriveGeneric #-}
{-# LANGUAGE LambdaCase #-}
{-# LANGUAGE TypeOperators #-}

module Main where

import           Data.Aeson
import           GHC.Generics
import           Network.Wai.Handler.Warp
import           Servant
import           System.IO
import           Servant.Swagger.UI
import           Servant.Swagger
import           Data.Swagger

data In = In { time :: String } deriving (Show, Eq, Generic)
instance FromJSON In
instance ToSchema In
data Out = Out { status :: String } deriving (Show, Eq, Generic)
instance ToJSON Out
instance ToSchema Out

type TestApi = "foo" :> ReqBody '[JSON] In :> Post '[JSON] Out
type SwaggerApi = SwaggerSchemaUI "swagger-ui" "swagger.json"
                :<|> TestApi

swaggerApi :: Proxy SwaggerApi
swaggerApi = Proxy

main :: IO ()
main = do
  let port = 3000
      settings =
        setPort port $
        setBeforeMainLoop (hPutStrLn stderr ("listening on port " ++ show port)) $
        defaultSettings
  runSettings settings =<< test

test :: IO Application
test = pure $ serve swaggerApi $
  swaggerSchemaUIServer (toSwagger (Proxy :: Proxy TestApi))
  :<|> const (pure $ Out "OK")
```
I don't see relevant error messages in the browser console, just no request gets issued when trying to do so using Swagger UI.

Tried it with recent hackage versions of servant and swagger packages (swagger2 2.1.4.1, servant 0.11, servant-server 0.11, servant-swagger 1.1.3.1, servant-swagger-ui 0.2.4.3.0.20).

**Comment by ZjMNZHgG5jMXw** [comment]
Same here. :(

**Comment by phadej** [comment]
Fixed in dist/servant-swagger-ui-0.2.4.3.4.0.tar.gz

**Comment by ZjMNZHgG5jMXw** [comment]
Works for me, thanks! :)
